### DISCUSSION TITLE
[BUG] 500 Error when accessing Dashboard

### DISCUSSION THREAD

**Discussion by voyager2009** [nextgenhealthcare/connect#5453] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89126292/193987631-19701b1f-be7f-4423-a865-b73e2dad52e5.png)
After installation (**mirthconnect-4.1.1.b303-windows-x64**), when I try to access Dashboard after launching I see above error. It doesn't go away and keeps popping up even after closing the dialog box ? Thanks for help !!!!!!!!!!!!

**Comment by ab-20-m** [comment, marked as answer]
See discussion [5392](https://github.com/nextgenhealthcare/connect/discussions/5392). Looks like Mirth is running under Java 9+ without having applied the [required extra steps](https://github.com/nextgenhealthcare/connect#5-running-nextgen-connect-in-java-9-or-greater).

**Comment by jonbartels** [comment]
Off topic but I wanted to compliment @ab-20-m . I see you answering questions regularly, thank you for helping! If you have time, come say hello in Slack.

**Comment by pacmano1** [comment]
Move to discussions until @voyager2009 can replicate the problem post @ab-20-m suggestions.

**Reply by voyager2009** [reply]
Working now. Thanks
